python/eos/tasks.py

```diff
@@ -259,8 +256,8 @@ def find_clusters(posterior, base_directory='./', threshold=2.0, K_g=1, analysis
     eos.data.MixtureDensity.create(os.path.join(base_directory, posterior, 'clusters'), density)
 
 
-@task('{posterior}/product')
-def mixture_product(posterior, posteriors, base_directory='./', analysis_file=None):
+@task('mixture-product', '{posterior}/product')
+def mixture_product(posterior:str, posteriors:list, base_directory:str='./', analysis_file:str=None):
```

Comment:
Unrelated to your commit, but why do I kept two arguments, `posterior` and `posteriors`? `posterior` could be the first element of posteriors by convention and one would do:
```
mixture_product( [postA, postB] )
```
instead of 
```
mixture_product( postA, [postB] )
```
?

Reply:
I think your example makes no sense, since that would not be a mixture product. The following example makes more sense:
```python
mixture_product('WET+FF', ['WET', 'FF'], ...)
```